design/001-RPC-Interface-Specification.md

```diff
@@ -40,8 +40,8 @@ The node shall provide three sets of APIs:
 2. Session APIs - For accessing the session related functionality. Each request
    should include a Session ID. Following APIs should be provided:
 
-    1. [Add Contact](#1-add-contact)
-    2. [Get Contact](#2-get-contact)
+    1. [Add PeerID](#1-add-peerID)
+    2. [Get PeerID](#2-get-peerID)
```

Comment:
```suggestion
    1. [Add PeerID](#1-add-peer-id)
    2. [Get PeerID](#2-get-peer-id)
```

Reply:
Since the heading is renamed below, the anchor name also is updated accordingly. Also not `id` should be used as lower case, an anchor created from a heading of any level is all lower case in Github Markdown.

Also I am collecting a suggestion to one other line that which I am not able to comment on:
1. Line 66: "1. Peer" to "1. Peer ID"

One more suggestions not in the scope of this PR, yet fixes a wrong link:
1. Lines 176, 258, 279: `(3-payment-channel-info)` to `(#3-payment-channel-info)`